app/src/main/java/com/example/pinterbest/MainActivity.kt

```diff
@@ -0,0 +1,105 @@
+package com.example.pinterbest
+
+import android.annotation.SuppressLint
+import android.os.Bundle
+import android.view.MenuItem
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
+import androidx.navigation.NavController
+import androidx.navigation.fragment.NavHostFragment
+import androidx.navigation.ui.setupWithNavController
+import com.example.pinterbest.databinding.ActivityMainBinding
+import com.google.android.material.navigation.NavigationView
+
+class MainActivity : AppCompatActivity(), NavigationView.OnNavigationItemSelectedListener {
+    private lateinit var binding: ActivityMainBinding
+
+    private var _navHostFragment: NavHostFragment? = null
+    private val navHostFragment get() = _navHostFragment!!
+
+    private var _navController: NavController? = null
+    private val navController get() = _navController!!
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        setTheme(R.style.Theme_Pinterbest)
+        super.onCreate(savedInstanceState)
+        binding = ActivityMainBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        // Устанавливаем BottomNavigationBar
```

Comment:
looks like useless comment. Overall, try to stick to approach of self documenting code

Reply:
Fixed here: https://github.com/enginire4ek/Pinterbest/pull/1/commits/c349b66d96a4b6bbf03557b4bc3771de1638810b